geoengine/types.py

```diff
@@ -138,13 +138,47 @@ def srs(self) -> str:
 
         return self.__srs
 
+    def __dict__(self) -> Dict[str, Any]:
```

Comment:
we would need to distinguish between raster and vector query rectangle

Reply:
24f3e00